src/hyperlight_host/src/func/guest_err.rs

```diff
@@ -31,7 +29,8 @@ pub(crate) fn check_for_guest_error(mgr: &MemMgrWrapper<HostSharedMemory>) -> Re
         ErrorCode::NoError => Ok(()),
         ErrorCode::OutbError => match mgr.as_ref().get_host_error()? {
             Some(host_err) => {
-                increment_guest_error_count(&guest_err);
+                metrics::counter!(METRIC_GUEST_ERROR, METRIC_GUEST_ERROR_LABEL_CODE => (guest_err.code as u64).to_string()).increment(1);
```

Comment:
In here, we are technically getting a host error. Is it worth considering adding an extra "host error" metric?

Reply:
Yeah, I'm doing part of that in #297 . We can reconvene then 👍